static/styles/portico/portico-signin.scss

```diff
@@ -326,11 +326,6 @@ html {
     margin-top: 25px;
 }
 
-.register-page .register-form .bottom-text {
```

Comment:
Can you make the commit title a bit more detailed and describe that you're only removing unused register page CSS